<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab12_PNR</class>
 <widget class="QMainWindow" name="Lab12_PNR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab12_PNR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblNumerator_PNR">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Numerator:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSub_PNR">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl1_2_PNR">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit1_PNR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lbl2_2_PNR">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNum2_PNR">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>210</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDiv_PNR">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>320</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd_PNR">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl2_1_PNR">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDenom2_PNR">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMult_PNR">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>320</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDenom1_PNR">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblDenominator_PNR">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Denominator:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNum1_PNR">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl1_1_PNR">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInverse_PNR">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>1/x</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radBtnNum1_PNR">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Num 1</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radBtnNum2_PNR">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>380</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Num 2</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
